lib/galaxy/tool_util/linters/datatypes.py

```diff
@@ -0,0 +1,84 @@
+import os.path
+from typing import (
+    Set,
+    TYPE_CHECKING,
+)
+
+from galaxy import config
+from galaxy.tool_util.lint import Linter
+from galaxy.util import (
+    listify,
+    parse_xml,
+)
+
+if TYPE_CHECKING:
+    from galaxy.tool_util.lint import LintContext
+    from galaxy.tool_util.parser import ToolSource
+
+DATATYPES_CONF = os.path.join(os.path.dirname(config.__file__), "sample", "datatypes_conf.xml.sample")
```

Comment:
Alternative options is packaging the sample file with something like https://setuptools.pypa.io/en/latest/userguide/datafiles.html or passing a callback back up to planemo that would manage downloading the file from the github repo.